ui/pages/bridge/transaction-details/transaction-details.tsx

```diff
@@ -0,0 +1,355 @@
+  const bridgeHistoryItem = srcTxHash ? bridgeHistory[srcTxHash] : undefined;
+  const networkConfigurationsByChainId = useSelector(
+    getNetworkConfigurationsByChainId,
+  );
+  const { srcNetworkConfiguration, destNetworkConfiguration } =
+    useBridgeChainInfo({
+      bridgeHistoryItem,
+    });
+
+  const srcBlockExplorerUrl = getBlockExplorerUrl(
+    srcNetworkConfiguration,
+    srcTxHash,
+  );
+
+  const destTxHash = bridgeHistoryItem?.status.destChain?.txHash;
+  const destBlockExplorerUrl = getBlockExplorerUrl(
+    destNetworkConfiguration,
+    destTxHash,
+  );
+
+  const srcChainTxMeta = selectedAddressTxList.find(
+    (tx) => tx.hash === srcTxHash,
+  );
+
+  const status = bridgeHistoryItem?.status.status;
+
+  const destChainIconUrl = destNetworkConfiguration
+    ? CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[
+        destNetworkConfiguration.chainId as keyof typeof CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP
+      ]
+    : undefined;
+  const bridgeTypeDirection = t('bridgeTypeDirectionTo');
+  const bridgeTypeDestNetwork = destNetworkConfiguration?.name;
+
+  const data = srcChainTxMeta
+    ? getTransactionBreakdownData({
+        state: rootState as MetaMaskReduxState,
+        transaction: srcChainTxMeta,
+        isTokenApprove: false,
+      })
+    : undefined;
+
+  const bridgeAmount = bridgeHistoryItem
+    ? `${calcTokenAmount(
+        bridgeHistoryItem.quote.srcTokenAmount,
+        bridgeHistoryItem.quote.srcAsset.decimals,
+      ).toFixed()} ${bridgeHistoryItem.quote.srcAsset.symbol}`
+    : undefined;
+
+  return (
+    <div className="bridge">
+      <div className="bridge__container">
+        <Header
+          className="bridge__header"
+          startAccessory={
+            <ButtonIcon
+              iconName={IconName.ArrowLeft}
+              size={ButtonIconSize.Sm}
+              ariaLabel={t('back')}
+              onClick={() => history.goBack()}
+            />
+          }
+        >
+          {t('bridge')} details
+        </Header>
+        <Content className="bridge__content">
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            gap={4}
+          >
+            {status !== StatusTypes.COMPLETE && bridgeHistoryItem && (
+              <BridgeStepList
+                bridgeHistoryItem={bridgeHistoryItem}
+                srcChainTxMeta={srcChainTxMeta}
+                networkConfigurationsByChainId={networkConfigurationsByChainId}
+              />
+            )}
+
+            {/* Links to block explorers */}
+            <BridgeExplorerLinks
+              srcBlockExplorerUrl={srcBlockExplorerUrl}
+              destBlockExplorerUrl={destBlockExplorerUrl}
+            />
+
+            <Divider />
+
+            {/* General tx details */}
+            <Box
+              display={Display.Flex}
+              flexDirection={FlexDirection.Column}
+              gap={2}
+            >
+              <TransactionDetailRow
```

Comment:
I think you need to use a smaller font size here to match the mocks

Reply:
Done in [16c6f64](https://github.com/MetaMask/metamask-extension/pull/27740/commits/16c6f643e196178b51ae0ac6318dd577d6d2bef6)